### ISSUE TITLE
custom_scalars‘ figure can't  be enlarged

### ISSUE THREAD

**Issue by yllgl** [lanpa/tensorboardX#613]
**Describe the bug**
I run the example code [demo_custom_scalars.py](https://github.com/lanpa/tensorboardX/blob/34d1616c035faaa0f3f7c9d19cb8bb4425f19939/examples/demo_custom_scalars.py) in my computer. However, when I click on the zoom button, the picture‘s width becomes smaller.

**Screenshots**
![Snipaste_2021-02-26_14-45-59](https://user-images.githubusercontent.com/23625282/109265478-969cd500-7841-11eb-9128-b90afd111fb0.png)

**Comment by lanpa** [comment]
Nice finding! Does the graph look normal before clicking the zoom button?

**Comment by yllgl** [comment]
> Nice finding! Does the graph look normal before clicking the zoom button?

Yes, it does.

**Comment by lanpa** [comment]
Thanks for the confirmation. I have also reproduced this issue and I think that it is caused by the TensorBoard frontend. 
In order to make sure that the bug is not caused by TBX, I have tried running the official custom scalar code
https://github.com/tensorflow/tensorboard/blob/50e0555f98c14a3aabcef7225ff9c72a0eea4173/tensorboard/plugins/custom_scalar/custom_scalar_demo.py but I can't run it properly. Once this issue is isolated, we can file a bug in the `tensorflow/tensorboard` repo.
